modules/ROOT/pages/anypoint-mq/2.x/anypoint-mq-consume.adoc

```diff
@@ -19,8 +19,8 @@ The operation above consumes the first available message in the queue identified
 * The message's content as payload.
 * The message's metadata in the message attributes.
 
-By default, the ACK mode is `MANUAL`, so the message isn't ACKed. If you want the message to be acknowledged as soon as it is consumed, set the mode to `IMMEDIATE`.
-For more information regarding a message ACK, see xref:anypoint-mq/anypoint-mq-ack.adoc[How to Handle Message Acknowledgment].
+By default, the ACK mode is `MANUAL`, so the message isn't acknowledged. If you want the message to be acknowledged as soon as it is consumed, set the mode to `IMMEDIATE`.
+For more information regarding a message ACK, see xref:anypoint-mq/2.x/anypoint-mq-ack.adoc[Anypoint MQ ACK and NACK Operations].
```

Comment:
regarding > about
"a message ACK" : is this a specific message acknowledgment, or ACKs in general?